swagger.yaml

```diff
@@ -18,17 +18,28 @@ consumes:
 produces:
   - text/plain
```

Comment:
why `text/plain` type for `consumes` and `produces`? In addition, the sections of `info`, `host`, and `basePath` should be updated as well.